tests/sqlparser_clickhouse.rs

```diff
@@ -826,6 +827,27 @@ fn parse_create_table_on_commit_and_as_query() {
     }
 }
 
+#[test]
+fn test_alter_table_with_on_cluster() {
```

Comment:
we can move the test similar to the create stmt over to common? since the current code applies to all dialects?

Reply:
Good call, will move it soon.